dev/bench/index.html

```diff
@@ -37,9 +37,15 @@
 </head>
 <body>
 <h1>Finch Benchmark</h1>
-<a href="./macOS/12/arm64/index.html">macOS 12 arm64</a>
-<a href="./macOS/12/x86_64/index.html">macOS 12 x86_64</a>
 <a href="./macOS/13/arm64/index.html">macOS 13 arm64</a>
 <a href="./macOS/13/x86_64/index.html">macOS 13 x86_64</a>
+<a href="./macOS/14/arm64/index.html">macOS 14 arm64</a>
+<a href="./macOS/14/x86_64/index.html">macOS 14 x86_64</a>
+<br>
+<details>
```

Comment:
We can alternatively just remove any reference to it. We can just get to the data if we need to by changing the URL manually.